x/auth/types/account.go

```diff
@@ -218,11 +218,6 @@ func (ma ModuleAccount) SetPubKey(pubKey cryptotypes.PubKey) error {
 	return fmt.Errorf("not supported for module accounts")
 }
 
-// SetSequence - Implements AccountI
-func (ma ModuleAccount) SetSequence(seq uint64) error {
```

Comment:
why was this like this in the first place? @amaurym @robert-zaremba do you know?

Reply:
afaik because the account didn't need replay protection

Reply:
I don't know. We don't check sequence number for module accounts. It seams it was added to make the `ModuleAccount` compatible / implementing the `auth/types.AccountI` interface

Reply:
This is an old code. It seams it was originally implemented by @fedekunze  ;) 352678438c8eadfae57e3314ce5a292ace0490a6